tests/cli/test_cli_parser.py

```diff
@@ -423,7 +423,7 @@ def test_invalid_choice_raises_for_export_format_in_db_export_archived_command(
         assert error_msg == (
             "\nairflow db export-archived command error: argument "
             f"--export-format: invalid choice: '{export_format}' "
```

Comment:
```suggestion
        assert error_msg.replace("'", "") == (
            "\nairflow db export-archived command error: argument "
            f"--export-format: invalid choice: {export_format}"
```

As Jarek, mentioned in slack this is happening for 3.12, so may be we can do like this then it should work for all versions, know this is bit ugly but dont have any other option :)

Reply:
I dont love this idea but I do not have a better one too.
I am thinking instead, we just add  a check for PY3.12+ and check?

@potiuk @gopidesupavan wdyt?

Reply:
Maybe:

```
assert "airflow db export-archived command error: argument --export-format: invalid choice" in error_msg
```

We do not have to test for exact equality

Reply:
Yeah good idea, thanks.